integration/registry_test.go

```diff
@@ -0,0 +1,238 @@
+// +build integration
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	testcontainers "github.com/testcontainers/testcontainers-go"
```

Comment:
NIT: This shows as redundant alias in my IDE

Reply:
Yes, I know. But my IDE repeatedly inserts and deletes the import if I don't add `testcontainers`. It's weird, but I have no idea to fix this issue...
![goland](https://user-images.githubusercontent.com/2253692/78576786-d75cbb80-7835-11ea-928e-8ab220319e9a.gif)

Reply:
FWIW @danielpacak  and @knqyf263, I have the same behaviour as seen here. I tried very hard to get rid of it too but it won't. What IDE are you running @danielpacak? What version?

Reply:
> FWIW @danielpacak and @knqyf263, I have the same behaviour as seen here. I tried very hard to get rid of it too but it won't. What IDE are you running @danielpacak? What version?

I'm using Intellij IDEA ULTIMATE 2019.3.4 with JetBrains Go plugin v193.6494.35.125

Reply:
I guess the behaviour is different across IDEs. I'm using Goland v2019.3.4